tests/neo4j/test_tx_func_run.py

```diff
@@ -202,14 +202,20 @@ def update2(tx):
             with self.assertRaises(types.DriverError) as e:
                 tx.run("MATCH (a:Node) SET a.property = 2").consume()
             exc = e.exception
-            if (exc.code
+            # TODO REMOVE THIS BLOCK ONCE ALL IMPLEMENT RETRYABLE EXCEPTIONS
+            if (get_driver_name() in ["javascript", "ruby", "python"]
```

Comment:
May I suggest to not put Ruby in there? Once the community driver implements the 5.0 changes, I think it should be notified of this required change through a failing pipeline.